scanpy_scripts/cmd_options.py

```diff
@@ -68,6 +68,14 @@
             'If not empty and not ending with "/" or "_", a "_" will be '
             "appended.",
         ),
+        click.option(
+            "--mtx-compression",
+            "-G",
+            type=click.Choice(["zip", "gzip", "bz2", "zstd"]),
+            default=None,
```

Comment:
should the default option be one of the compressed options?

Reply:
I think it's okay to have this set to `None` since the underlying function to generate the matrix accepts `None` (no compression) and will not return an error. But you're right that it might be a good idea feature-wise, if a compressed matrix is more commonly used directly downstream, e.g., direct downstream tools can accept compressed files (?).

Reply:
Got that!